.pre-commit-config.yaml

```diff
@@ -124,7 +124,7 @@ repos:
       - id: prettier
         types: [yaml]
   - repo: https://github.com/qarmin/qml_formatter.git
-    rev: 37c2513b1b8275a475a160ed2f5b044910335d5f # No release tag yet including #6 fix
+    rev: latest # No release tag yet including #6 fix
```

Comment:
this should be reverted... pre-commit does that automatically unfortunately when running `autoupdate`.

Reply:
I opened PR https://github.com/mixxxdj/mixxx/pull/13880, which reverts this unintended change.